modules/users/client/views/admin/view-applicants.html

```diff
@@ -27,9 +27,9 @@
         <tbody>
             <tr class="applicant-rows" ng-repeat="applicant in vm.unapprovedUsers">
              	<td>{{ applicant.firstName }} {{ applicant.lastName }}</td>
-             	<td>{{ applicant.roles }}</td>
-                <td class="btn btn-success" ng-click="vm.approveUser(applicant)">√</td>
```

Comment:
I also liked the look of the check and x. I think it's a minor detail but as long as it's semantically clear what the view is for, the symbols are cleaner.